### PR TITLE
favourites bug

### PR DESCRIPTION
**Pull request by philyeung**

### CODE REVIEW

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import useMoviesContext from "../context/MoviesContext";
 
 export const Tabs = Object.freeze({
   Movies: "Movies",
-  Favourites: "Favorites",
+  Favourites: "Favorits",
```
Comment (KushTurner) [inline src/components/NavBar.jsx]: https://github.com/nfeagh/react-flask-app/blob/main/inbox/email_J_10.md Can you please also update this as well as it is no longer to do with American spelling.

File: src/pages/MainPage.jsx [hunk 2]
```diff
@@ -39,13 +39,14 @@ export default function MainPage() {
             ? movies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
             : null}
           {currentTab === Tabs.Favourites && favouriteMovies?.length
-            ? favouriteMovies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
+            ? recommendedMovies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
```
Comment (KushTurner) [inline src/pages/MainPage.jsx:41]: Are we not going to add a task for this? Should we let the students figure it out themselves? Would be interesting lol.

File: src/components/NavBar.jsx (hunk 1 above)
Comment (philyeung) [reply src/components/NavBar.jsx]: ah actually that's a good point I forgot about this...
Ok let's leave your email in, let me back this one out and we can discuss with Jeenal on Tuesday.

File: src/pages/MainPage.jsx (hunk 2 above)
Comment (philyeung) [reply src/pages/MainPage.jsx:41]: Yeah so this one we're gonna see if they can spot the bug - we wanted them to be able to raise some defects of their own

**Review by KushTurner — APPROVED** [review]

**Pull request merged**